templates/express_mysql/.env

```diff
@@ -0,0 +1,4 @@
+HOST=
+USER=
+PASSWORD=
+DATABASE=
```

Comment:
> [!Tip]
> Use very clear (medium length) env var names. eg. DB_USER, DB_PASSWORD, ... etc

`USER` is a reserved env var name in linux based OS. 
Creates a confusion and because of the value here will not be read!!

https://www.linuxtopia.org/online_books/introduction_to_linux/linux_Reserved_variables.html